test/unit/lodash.spec.js

```diff
@@ -0,0 +1,709 @@
+            });
+        });
+
+        describe('get2', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.get2();
+            });
+
+            it('should check types', function(){
+                var elem = params[1];
+
+                expect(elem instanceof Array).to.eql(true);
+                expect(elem).to.have.length(4);
+            });
+            it('should return the resolved value from object', function(){
+
+                var month = ['January', 'February', 'March', 'Aprlil', 'May'],
+                    day = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Friday'];
+
+                var randMonth = function(){
+                    return _.random(0, 4);
+                };
+
+                var randNameDay = function(){
+                    return _.random(0, 6);
+                };
+
+                var randDay = function(){
+                    return _.random(1,31);
+                };
+
+                var obj = {
+                    month: month[randMonth()],
+                    nameDay: day[randNameDay()],
+                    day: randDay(),
+                    exercises: {tasks: [{firstTask: 'write a code'},{secondTask: 'learn lodash'}, {thirdTask: 'be cool'}]}
+                };
+                expect(_.get.apply(_, datasets.get2(obj))).to.eql('be cool');
+            });
+        });
+
+
+        describe('get3', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.get3();
+            });
+
+            it('should check types', function(){
+                var elem2 = params[1],
+                    elem3 = params[2];
+
+                expect(typeof elem2).to.eql('string');
+                expect(typeof elem3).to.eql('string');
+            });
+            it('should return the resolved value from object', function(){
+
+                var month = ['January', 'February', 'March', 'Aprlil', 'May'],
+                    day = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Friday'];
+
+                var randMonth = function(){
+                    return _.random(0, 4);
+                };
+
+                var randNameDay = function(){
+                    return _.random(0, 6);
+                };
+
+                var randDay = function(){
+                    return _.random(1,31);
+                };
+
+                var obj = {
+                    month: month[randMonth()],
+                    nameDay: day[randNameDay()],
+                    day: randDay(),
+                    exercises: {tasks: [{firstTask: 'write a code'},{secondTask: 'learn lodash'}, {thirdTask: 'be cool'}]}
+                };
+                expect(_.get.apply(_, datasets.get3(obj))).to.eql('default value');
+            });
+        });
+    });
+
+
+    describe('invert', function () {
```

Comment:
What exactly is tested in this case? Because what you want is to basically pass a random object from a param to dataset expectations, how is that testing developer knowledge of this function at all?